src/main/java/com/csc366/project/entity/Order.java

```diff
@@ -0,0 +1,147 @@
+package com.csc366.project.entity;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.Date;
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+
+@Entity
+@Table(name = "order")
+public class Order {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long orderId;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "id", nullable = true)
+    private Location location;
```

Comment:
I think in order for this to work we'll need an `Order` property and `@OneToMany` on the `Location` class

Reply:
agreed should be in there now